mappings/net/minecraft/client/render/entity/model/OcelotEntityModel.mapping

```diff
@@ -52,6 +52,7 @@ CLASS net/minecraft/unmapped/C_zcwxisym net/minecraft/client/render/entity/model
 	FIELD f_peuqruka HEAD_PIVOT_Y F
 	FIELD f_qtcudsvu FRONT_LEG_PIVOT_Z F
 	FIELD f_quaroerh LOWER_TAIL_PIVOT_Y F
+	FIELD f_rltdlggx X F
```

Comment:
This name isn't very descriptive - it is `X0` in Mojmap, while `Y0` is `BODY_SIZE_Y` in QM